.pylintrc

```diff
@@ -4,6 +4,11 @@
 # there seems to be an off by one issue in how max-nested-blocks is used.
 max-nested-blocks = 2
 
+[FORMAT]
```

Comment:
i wasn't sure if it was added by mistake. not sure what David's initial pull request is.